ibm_mq/datadog_checks/ibm_mq/utils.py

```diff
@@ -13,3 +18,29 @@ def sanitize_strings(s):
     if found >= 0:
         s = s[:found]
     return s.strip()
+
+
+def calculate_elapsed_time(datestamp, timestamp, current_time=None):
+    """
+    Calculate elapsed time in seconds from IBM MQ queue status date and timestamps
+    Assume queue manager uses local system timezone.
```

Comment:
the Agent is unlikely to run on the same system

Reply:
Good point, I've added a configurable `queue_manager_timezone` option that defaults to `UTC`.